frontend/components/Header.tsx

```diff
@@ -46,7 +46,7 @@ const Header = () => {
           </Link>
         </div>
         <div className="flex items-center justify-end space-x-2">
-          <div className="flex w-[200px] justify-end">
+          <div className="solana-wallet-button flex w-[200px] justify-end [&_button]:!rounded-none">
```

Comment:
Why !rounded_none, I'm curious

Reply:
LOL seems like it's to give more priority to the style and override another rule

Reply:
lets not do it this way, this is not very friendly to future maintainers, we can just override the solana-wallet-button class in globals.css